UnitsNet.Tests/UnitsNet.Tests.csproj

```diff
@@ -50,7 +50,105 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="obj\**" />
+    <Compile Include="CustomCode\AccelerationTests.cs" />
```

Comment:
You should only have the line `<Compile Include="**\*.cs" Exclude="obj\**" />`, all the other lines you added should be removed. If you take a look at the total changes of this PR, you will see a lot of changes to .csproj files. All these changes must be reverted to all .csproj files.

Make sure to close and re-open the solution so that Visual Studio picks up the changes, that is probably the compile error you got.